src/IndexManager.php

```diff
@@ -11,6 +11,8 @@
 
 class IndexManager implements IndexManagerInterface
 {
+    public $propertyAccessor;
```

Comment:
This must be declared public. Not the ideal, but otherwise  would be a breaking change.

Reply:
One could argue that since it was never documented it can be considered an internal API and thus made `protected`. On the other hand, people may have been relying on it. This is a tricky one.

Reply:
Damit 😭 
I tend to agree with @alcaeus that we could consider it internal and use protected. I doubt that anyone used it externally. Worth case scenario, we make it public in a following patch release.